Makefile

```diff
@@ -118,6 +120,52 @@ golint: go-lint
 ## docs: Generate swagger docs.
 docs: go-gen-docs
 
+## install-newman: Install Postman Newman for tests.
+install-newman:
+	@echo "  >  Running Postman Newman"
+ifeq (,$(shell which newman))
+	@echo "  >  Installing Postman Newman"
+	@-brew install newman
+endif
+
+## run-newman: Run Postman Newman tests.
+run-newman: install-newman newman-txs newman-tokens newman-staking newman-collections newman-domains newman-observer newman-healthcheck
+
+## newman-txs: Run Newman tests for Transaction API's.
+newman-txs:
```

Comment:
lots of code duplication. 

Did you consider improving interface to run tets?

`make integrations tokens`
`make integrations staking`

 ? or anything similar

It's pointless to create each command for specifying each file